message_hub/src/main/java/ordina/youbionic/presentation/ServoController.java

```diff
@@ -0,0 +1,37 @@
+package ordina.youbionic.presentation;
+
+import ordina.youbionic.infrastructure.QueueEnum;
+import ordina.youbionic.infrastructure.RabbitMQPublisher;
+import ordina.youbionic.service.ServoService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ServoController {
+    private final ServoService service;
+
+    public ServoController(){
+        this.service = new ServoService();
+    }
+
+    @GetMapping("/test0")
+    public String test0() throws Exception{
+        return service.test0();
+    }
+
+    @GetMapping("/test1")
+    public String test1() throws Exception{
+        RabbitMQPublisher publisher = new RabbitMQPublisher();
```

Comment:
The controller should only be calling a service and should have as little logic as possible